src/ui/zcl_abapgit_gui_page_diff.clas.abap

```diff
@@ -739,12 +773,14 @@ CLASS zcl_abapgit_gui_page_diff IMPLEMENTATION.
 
     " New line
     lv_mark = ` `.
-    IF iv_fstate = c_fstate-both OR is_diff_line-result = zif_abapgit_definitions=>c_diff-update.
-      lv_bg = ' diff_upd'.
-      lv_mark = `~`.
-    ELSEIF is_diff_line-result = zif_abapgit_definitions=>c_diff-insert.
-      lv_bg = ' diff_ins'.
-      lv_mark = `+`.
+    IF is_diff_line-result IS NOT INITIAL.
```

Comment:
this seems like logical changes to the diff page, is this needed in relation to zip and offline repos?

Reply:
No, this is the mentioned `bugfixed diff page - yellowed all the lines when MM status`
For MM files all diff chunk was yellow, not only the changed lines. Maybe even my bug originally, but I don't remember :)

Reply:
yeah, resolve, note that this could have been in a separate PR, as this is a different feature/bugfix